tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -253,3 +253,44 @@ Feature: add users to group
     And user "Brian" has been added to group "music"
     When the administrator "Alice" tries to add a group "music" to a group "student" using the Graph API
     Then the HTTP status code should be "400"
+
+
+  Scenario Outline: try to add invalid user id to a group
```

Comment:
```suggestion
  Scenario Outline: try to add user with invalid id to a group
```

Reply:
https://github.com/owncloud/ocis/pull/5864#discussion_r1144210941

Reply:
A user cannot have an invalid id and `add user with invalid id` could lead to misunderstanding that there can be a valid user with invalid id pattern